src/constants.ts

```diff
@@ -39,6 +39,10 @@ export const availableVaultsPerChain: Record<Chain, `0x${string}`[]> = {
   [Chain.POLYGON]: [],
   [Chain.ARBITRUM]: [],
   [Chain.BASE]: [],
+  [Chain.ARBITRUM_SEPOLIA]: [
```

Comment:
These values should match the values in [src/collateralByNetwork.ts here](https://github.com/Parifi/references/pull/2/files#diff-8da9b13c9c988c1d7b70845f596691d2c67093d75c9536f763bd69ae4eacd4aeR49), no?

Reply:
done

Reply:
picked these values - 
  "vaults": {
    "pfUSDC": "0x609f4BBbB50E06803F9C30e13057eC8746cafc82",
    "pfWETH": "0x32584Dc865c42c7107e9fe03FC0E9181f68458F5"
  },
  
  https://www.notion.so/parifi/083616b7a4bf4b3bba832e1e27820601?v=66ac035c84fe43b1be4aa0a9b9ab7e62&p=79943ac25b024699bcdd48a716831ccc&pm=s

Reply:
My bad, got confused because of the comment. Thought this to be ETH collateral. Ignore the comment, we can merge